nnpdf_data/nnpdf_data/commondata/CDF_Z0_1P96TEV/metadata.yaml

```diff
@@ -5,21 +5,21 @@ nnpdf_metadata:
   nnpdf31_process: DY NC
   experiment: CDF
 arXiv:
-  url: https://arxiv.org/abs/908.3914
+  url: https://arxiv.org/abs/0908.3914
   journal: Phys. Lett. B692 (2010) 232
 iNSPIRE:
-  url: ''
+  url: https://inspirehep.net/literature/856131
 hepdata:
-  url: ''
-  version: -1
+  url: https://www.hepdata.net/record/ins856131?version=1&table=Table%202
+  version: 1
 implemented_observables:
 - observable_name: ZRAP
   observable:
     description: Drell-Yan Rapidity Distribution
     label: CDF $Z$ rapidity (new)
     units: ''
   process_type: EWK_RAP
```

Comment:
Hi @jacoterh 

Remember to put as `process_type:` one of the processes that you will find in this file: https://github.com/NNPDF/nnpdf/blob/master/validphys2/src/validphys/process_options.py (at the bottom, either DY_Z_Y or DY_W_ETA)

and also put `kinematic_override: identity`.

This is to avoid going through some old code that we are trying to remove.

When you do that you will see that that file defines which kinematic variables are allowed, sometimes it is necessary to change from Q -> Q^2 or pT -> pT^2 (the kinematic info in the end is the same,  but we are trying to standardize it a bit more)

Reply:
Ok, done!